### DISCUSSION TITLE
flatpicker breaks when after building for production

### DISCUSSION THREAD

**Discussion by tweiss777** [ankurk91/vue-flatpickr-component#268] [Q&A] [answered]
When working with vue-flatpickr-component in development everything is working fine. When building our vue project for production and running it on docker or deploying it to AWS the library breaks and we see the calendars all over the page shown in the screenshot below.
<img width="691" alt="Screenshot 2022-11-24 at 16 57 53" src="https://user-images.githubusercontent.com/23391038/203814471-5bb485d9-f322-4385-9c06-2fa857fa2ed9.png">

We're using Vue2 and vue-flatpickr-component v8.1.6

Any help solving this issue would greatly be appreciated!

**Comment by ankurk91** [comment, marked as answer]
Haha, you are expecting me to dive into your deployment pipeline?

No one can help you without a minimal reproduction of code .

Check your css😉

**Reply by tweiss777** [reply]
Thanks we will check it out :)
